### DISCUSSION TITLE
need help for kml Layers

### DISCUSSION THREAD

**Discussion by cyril-cordier** [JustFly1984/react-google-maps-api#2242] [Q&A]
I do not understand the doc concerning the use of kml layers. I am unable to apply the example in my project. Is there anyone who has used this feature before?
Thank you

**Comment by cyril-cordier** [comment]
In reality, the error in the example seems to come from the kml link which is no longer valid and which should be replaced by the link from the official Google API doc :  https://googlearchive.github.io/js-v2-samples/ggeoxml/cta.kml

However, I cannot specify in the url of the KmlLayer component the link to a local kml file. Am I not doing it right?

![Capture d’écran 2021-03-10 170137](https://user-images.githubusercontent.com/65895103/110658863-597dfe80-81c2-11eb-9a12-4ebc232e69ee.png)

**Reply by JustFly1984** [reply]
I guess your bundler do not handle kml  extensions.
Put your file to public folder and provide full url, or try to find the way for your bundler to handle kml files.

**Reply by cyril-cordier** [reply]
Thanks for your feedback.
I tried to put the kml files in src with an import, the link becomes 'media / kml / filename.kml' but the KmlLayer does not work.
I tried putting the kml files in public, the link becomes process.env.PUBLIC_URL + '/kml/filename.kml' but the KmlLayer doesn't work either.
I found an alternative solution for storing kml files on cloudinary.com but I'm limited to 10MB per file, and I have kml files that are up to 170MB!
